stdlib/public/core/DictionaryStorage.swift

```diff
@@ -511,6 +505,3 @@ extension _DictionaryStorage {
     return storage
   }
 }
-
-extension _DictionaryStorage: @unchecked Sendable
```

Comment:
I believe this unchecked `Sendable` conformance was incorrect, and `_DictionaryStorage` inherits an unavailable `Sendable` conformance from `__RawDictionaryStorage`.

Reply:
Yeah I think you're right about this. Should be fine to remove 👍